pkgs/tools/system/fakeroot/default.nix

```diff
@@ -39,6 +39,9 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  postInstall = ''
+    sed -i -e "s#for dir in \`echo \$PATHS | sed 's/:/ /g'\`#for dir in \`echo \$PATHS | ${gnused}/bin/sed 's/:/ /g'\`#" $out/bin/fakeroot
```

Comment:
You could use a shorter.
```
sed -i -e "s#\<sed\>#${gnused}/bin/sed#g" $out/bin/fakeroot
```

Or you could even move it to `postUnpack`:
```
sed -i "s@sed@$(type -p sed)@g" ${name}/scripts/fakeroot.in
```

Reply:
@clefru ping

Reply:
I adopted the last solution. fakeroot.in seems much easier to patch, thanks for the suggestion.

Also I merged it with the @getopt@ patching, which had a weird no-op-ish for loop around it, which I removed.